fix(windows): changed instance size and added small docs (#31)
kumahq/envoy-builds: terraform/README.md

## Windows

### Create an instance

> Run:
> 
> ```bash
> terraform plan -out plan
> ```
> 
> After you confirm and validate that plan is correct, apply the plan:
> 
> ```bash
> terraform apply "plan"
> ```

Reviewer: ```suggestion
Run `terraform apply`, confirm that the plan is correct and apply it.
```
